src/askjsx/lib/Let.tsx

```diff
@@ -20,12 +25,13 @@ export function Set({
   );
 
   return (
-    <code let>
+    <code {...{ [code]: true }}>
```

Comment:
if code is just a string, isn't it enough to just print it here?

Reply:
I don't knowa shorter way to add a new dynamic property in JSX tag